pgmigrate.py

```diff
@@ -494,15 +502,16 @@ def _get_callbacks(callbacks, base_dir=''):
     return _parse_str_callbacks(callbacks, ret, base_dir)
 
 
-def _migrate_step(state, callbacks, base_dir, user, cursor):
+def _migrate_step(state, callbacks, base_dir, user, db_handle):
```

Comment:
`db_handle=(cursor, schema)` is a workaround to make pylint `too-many-arguments` check pass